iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/user.component.js

```diff
@@ -16,6 +16,30 @@
 (function () {
     'use strict';
 
+    function ResignAupController($scope, $uibModalInstance, AupService, user, toaster) {
```

Comment:
as said in the previous component, this controller would be moved in a separated file

Reply:
Done